packages/bsky/src/data-plane/server/db/tables/post.ts

```diff
@@ -15,6 +15,7 @@ export interface Post {
   tags: string[] | null
   invalidReplyRoot: boolean | null
   violatesThreadGate: boolean | null
+  violatesQuoteGate: boolean | null
```

Comment:
Coming from the dataplane this will be `violatesQuoteGate`. Everywhere else I refer to it as `violatesQuotegate`.